fastapi_users/authentication/cookie.py

```diff
@@ -46,6 +47,7 @@ def __init__(
         cookie_domain: str = None,
         cookie_secure: bool = True,
         cookie_httponly: bool = True,
+        cookie_samesite: str = 'lax',
```

Comment:
Could you call the `make format` command? It ensures the code is properly formatted with `black`.

Reply:
Will do! I have some work to take care of this morning. I'll try to get to this around lunchtime PST.

Reply:
Sorry for lagging on this! Just release alpha for my app yesterday so I've been caught up with bugfixes for that. I will get to this.

Reply:
Alright, I reformatted with black (using `make format`) it changed things in a lot of files though:
```
pipenv run isort ./fastapi_users
Courtesy Notice: Pipenv found itself running within a virtual environment, so it will automatically use that environment, instead of creating its own for any project. You can set PIPENV_IGNORE_VIRTUALENVS=1 to force pipenv to ignore that environment and create its own instead. You can set PIPENV_VERBOSITY=-1 to suppress this warning.
WARNING: Unable to parse file ./fastapi_users due to [Errno 21] Is a directory: '/home/user/dev/Yacht-work/fastapi-users/fastapi_users'
pipenv run isort ./docs/src -o fastapi_users
Courtesy Notice: Pipenv found itself running within a virtual environment, so it will automatically use that environment, instead of creating its own for any project. You can set PIPENV_IGNORE_VIRTUALENVS=1 to force pipenv to ignore that environment and create its own instead. You can set PIPENV_VERBOSITY=-1 to suppress this warning.
WARNING: Unable to parse file ./docs/src due to [Errno 21] Is a directory: '/home/user/dev/Yacht-work/fastapi-users/docs/src'
pipenv run black .
Courtesy Notice: Pipenv found itself running within a virtual environment, so it will automatically use that environment, instead of creating its own for any project. You can set PIPENV_IGNORE_VIRTUALENVS=1 to force pipenv to ignore that environment and create its own instead. You can set PIPENV_VERBOSITY=-1 to suppress this warning.
reformatted /home/user/dev/Yacht-work/fastapi-users/docs/src/full_sqlalchemy.py
reformatted /home/user/dev/Yacht-work/fastapi-users/docs/src/full_tortoise.py
reformatted /home/user/dev/Yacht-work/fastapi-users/docs/src/full_mongodb.py
reformatted /home/user/dev/Yacht-work/fastapi-users/docs/src/oauth_full_sqlalchemy.py
reformatted /home/user/dev/Yacht-work/fastapi-users/docs/src/oauth_full_mongodb.py
reformatted /home/user/dev/Yacht-work/fastapi-users/docs/src/oauth_full_tortoise.py
reformatted /home/user/dev/Yacht-work/fastapi-users/fastapi_users/authentication/jwt.py
reformatted /home/user/dev/Yacht-work/fastapi-users/fastapi_users/fastapi_users.py
reformatted /home/user/dev/Yacht-work/fastapi-users/fastapi_users/router/users.py
reformatted /home/user/dev/Yacht-work/fastapi-users/fastapi_users/router/oauth.py
reformatted /home/user/dev/Yacht-work/fastapi-users/tests/test_db_mongodb.py
reformatted /home/user/dev/Yacht-work/fastapi-users/tests/conftest.py
reformatted /home/user/dev/Yacht-work/fastapi-users/tests/test_db_sqlalchemy.py
reformatted /home/user/dev/Yacht-work/fastapi-users/tests/test_db_tortoise.py
reformatted /home/user/dev/Yacht-work/fastapi-users/tests/test_fastapi_users.py
reformatted /home/user/dev/Yacht-work/fastapi-users/tests/test_router_register.py
reformatted /home/user/dev/Yacht-work/fastapi-users/tests/test_router_oauth.py
reformatted /home/user/dev/Yacht-work/fastapi-users/tests/test_router_users.py
All done! ✨ 🍰 ✨
18 files reformatted, 28 files left unchanged.
```